build-apisix-openresty-centos7.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
Please let this script call `build-apisix-openresty.sh` instead of copying code from it.

Reply:
But I need to change the `. /configure` parameter in `build-apisix-openresty.sh`. 
Can I call it to change the contents?

Reply:
use `sed`

Reply:
update